.github/workflows/docker-build-test.yaml

```diff
@@ -200,7 +198,7 @@ jobs:
     with:
       GIT_SHA: ${{ needs.determine-docker-build-metadata.outputs.gitSha }}
       FORGE_TEST_SUITE: compat
-      IMAGE_TAG: 843b204dce971d98449b82624f4f684c7a18b991 # test against the latest build on testnet branch
+      IMAGE_TAG: testnet_2d8b1b57553d869190f61df1aaf7f31a8fc19a7b # test against the latest build on testnet branch
```

Comment:
Changes to this file in particular are the primary ones that will unblock testing on `mainnet` branch. (see this PR: https://github.com/aptos-labs/aptos-core/pull/5781#issuecomment-1340363591)

If we want to be more granular about these changes we're bringing in, we can just checkout these changes on `.github/workflows/docker-build-test.yaml`